### DISCUSSION TITLE
open-webui serve: error loading fbgemm.dll or one of its dependecies

### DISCUSSION THREAD

**Discussion by F041** [open-webui/open-webui#4516] [General]
# Bug Report

## Installation Method

I did it after installing aider-chat. In fact when I run 
pip install open-webui
I get
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
aider-chat 0.48.1 requires aiohttp==3.10.0, but you have aiohttp 3.9.5 which is incompatible.
aider-chat 0.48.1 requires packaging==24.1, but you have packaging 23.2 which is incompatible.
Successfully installed aiohttp-3.9.5 packaging-23.2
## Environment

- **Open WebUI Version:** latest
- **Ollama (if applicable):** latest
- **Operating System:** Win 11 Pro
- **Browser (if applicable):** Brave
**Confirmation:**

- [x] I have read and followed all the instructions provided in the README.md.
- [x] I am on the latest version of both Open WebUI and Ollama.
- [ ] I have included the browser console logs.
- [ ] I have included the Docker container logs.
- [ ] I have provided the exact steps to reproduce the bug in the "Steps to Reproduce" section below.

## Expected Behavior:

localhost running service
## Actual Behavior:

OSERROR
## Description

**Bug Summary:**
[Provide a brief but clear summary of the bug]

## Reproduction Details

**Steps to Reproduce:**
[Outline the steps to reproduce the bug. Be as detailed as possible.]

## Logs and Screenshots

**Browser Console Logs:**
[Include relevant browser console logs, if applicable]

**Docker Container Logs:**
[Include relevant Docker container logs, if applicable]

**Screenshots/Screen Recordings (if applicable):**
[Attach any relevant screenshots to help illustrate the issue]

## Additional Information

[Include any additional details that may help in understanding and reproducing the issue. This could include specific configurations, error messages, or anything else relevant to the bug.]

## Note

If the bug report is incomplete or does not follow the provided instructions, it may not be addressed. Please ensure that you have followed the steps outlined in the README.md and troubleshooting.md documents, and provide all necessary information for us to reproduce and address the issue. Thank you!

**Comment by justinh-rahb** [comment]
Use a VirtualEnv.
